app/components/find_interface/courses/about_schools_component/view.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-schools"><%= course.placements_heading %></h2>
+  <div data-qa="course__about_schools">
+    <% if course.program_type == "higher_education_programme" && course.provider.provider_code != "B31" %>
+      <%= render FindInterface::Utility::AdviceComponent::View.new(title: "Where you will train") do %>
+        <p class="govuk-body">You’ll be placed in schools for most of your course. Your school placements will be within
+          commuting distance.</p>
+        <p class="govuk-body">You can’t pick which schools you want to be in, but your university will try to take your
+          journey time into consideration.</p>
+        <p class="govuk-body">Universities can work with over 100 potential placement schools. Most will be within 10
+          miles of the university, but sometimes they can cover a wider area, especially outside of cities.</p>
+      <% end %>
+    <% elsif course.program_type == "scitt_programme" && course.provider.provider_code != "E65" %>
+      <%= render FindInterface::Utility::AdviceComponent::View.new(title: "Where you will train") do %>
+        <p class="govuk-body">You’ll be placed in different schools during your training. You can’t pick which schools
+          you want to be in, but your course will try to place you in schools you can commute to.</p>
+      <% end %>
```

Comment:
the block looks wrong

it should either be in the component (probably looks nicer) or

``` ruby

<%= render FindInterface::Utility::AdviceComponent::View.new(title: "Where you will train") do %>
     <% if course.program_type == "higher_education_programme" && course.provider.provider_code != "B31" %>
foo
     <% elsif course.program_type == "scitt_programme" && course.provider.provider_code != "E65" %>
bar
     <% else %>
foobar
<% end %>

     ```